src/sass/variables/_Font.Variables.scss

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+// Font name
+$ms-font-name: 'Segoe UI' !default;
+
+// Font directory and paths
+$ms-font-directory:         'https://static2.sharepointonline.com/files/fabric/assets/fonts/' !default;
+$ms-font-path-arabic:       'segoeui-arabic' !default;
+$ms-font-path-cyrillic:     'segoeui-cyrillic' !default;
+$ms-font-path-easteuropean: 'segoeui-easteuropean' !default;
+$ms-font-path-greek:        'segoeui-greek' !default;
+$ms-font-path-hebrew:       'segoeui-hebrew' !default;
+$ms-font-path-vietnamese:   'segoeui-vietnamese' !default;
+$ms-font-path-westeuropean: 'segoeui-westeuropean' !default;
+
+// Base font stacks
+$ms-font-system-base: 'Segoe UI', -apple-system, BlinkMacSystemFont, 'Roboto', 'Helvetica Neue', sans-serif !default;
+$ms-font-family-base: 'Segoe UI WestEuropean', $ms-font-system-base !default;
```

Comment:
No, 'Segoe UI' is the local system font ($ms-font-system-base fonts are all local system fonts). 'Segoe UI West European' is the friendly name for the web font. I'm assuming 'Segoe UI WestEuropean' would fail for Macs and non-Windows devices.